core/vm/eips.go

```diff
@@ -174,3 +175,19 @@ func opBaseFee(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]
 	scope.Stack.push(baseFee)
 	return nil, nil
 }
+
+// enable3855 applies EIP-3855 (PUSH0 opcode)
+func enable3855(jt *JumpTable) {
+	// New opcode
+	jt[PUSH0] = &operation{
+		execute:     opPush0,
+		constantGas: GasFastestStep,
```

Comment:
According to the spec it's `verylow` = 2 gas (`GasFastestStep` = 3 gas)
```suggestion
		constantGas: GasQuickStep,
```

Reply:
Good catch, fixed.